apps/src/p5lab/drawUtils.js

```diff
@@ -41,9 +41,9 @@ export function getTextWidth(p5, text, size) {
  * @param {Number} y
  * @param {Number} width
  * @param {Number} height
- * @param {Number} config.triangleSize
- * @param {Number} config.triangleTipX
- * @param {Number} config.rectangleCornerRadius
+ * @param {Number} config.tailSize
+ * @param {Number} config.tailTipX
+ * @param {Number} config.padding
```

Comment:
I think we might want to continue using the keyword "radius" in this variable - my understanding is that we still use it for the radius value in p5.rect and since padding has other context (in CSS), I'm hesitant to overload the meaning here.

Reply:
That's fair. It seemed silly to have two constants with the same value, but it's probably better if it helps with readability.